utils/testing/view_test_utils.py

```diff
@@ -53,9 +54,14 @@ def assertValidPostResponse(self, data=None, url=None, redirect_url=None, fetch_
         data = data or {}
         response = self.client.post(url, data=data)
         if redirect_url:
+            # If a form errors, it returns a templateresponse instead. So we can instantly debug
```

Comment:
Should probably add `utils/testing` to the `.coveragerc`: Coverage.py is picking up missed lines for testing utilities.